### DISCUSSION TITLE
SOCKS5 proxy in chrome command line configuration

### DISCUSSION THREAD

**Discussion by miquecg** [bromite/bromite#2138] [Q&A]
Hi,

I've noticed a problem with latest Bromite version 102.0.5005.96 on Android. I have a SOCKS5 proxy configured long time ago using a `chrome-command-line` file located in `/data/local`.

`_ --proxy-server="socks5://proxy-server.com" --host-resolver-rules="MAP * ~NOTFOUND , EXCLUDE proxy-server.com"`

It worked like a charm until now. Required `Enable command line on non-rooted devices` flag is enabled.

I tried to downgrade to the previous release but system doesn't allow me to install the apk, probably to protect user from downgrading apps.

**Comment by ghost** [comment]


**Reply by miquecg** [reply]
Thanks, first command did the trick. I just needed to space `-r -d` options.

Downgrading to 102.0.5005.92 version fixes the problem for me, so I guess there's some kind of issue with latest version.

**Reply by miquecg** [reply]
Hi @csagan5, would you mind if I open an issue with this or is there anything else I can check on my side?

**Reply by csagan5** [reply]
You should test with [Chromium](https://www.bromite.org/chromium) first (it is one of the questions asked when submitting issues), that is because if it is a Chromium issue you will have much better luck with seeing this fixed upstream (and there might be already an issue open there).

**Reply by miquecg** [reply]
Chromium 102.0.5005.96 tested and it works as usual.

I've created #2159 to track this issue.

Btw, promoting discussion to issue does not bring the template. I did it and got the first issue closed because of that.
